packages/components/menu/src/MenuSubheading/MenuSubheading.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { getMenuSubheadingStyles } from './MenuSubheading.styles';
+import { Subheading, SubheadingProps } from '@contentful/f36-typography';
+
+export type MenuSubheadingProps = SubheadingProps;
+
+export const MenuSubheading = (props: MenuSubheadingProps) => {
+  const {
+    children,
+    testId = 'cf-ui-menu-subheading',
+    className,
+    ...otherProps
+  } = props;
+
+  const styles = getMenuSubheadingStyles();
+
+  return (
+    <Subheading
+      testId={testId}
+      className={cx(styles, className)}
+      marginBottom="none"
+      {...otherProps}
+    >
+      {children}
+    </Subheading>
```

Comment:
Based on the designs this should be a SectionHeading

![Screenshot 2021-09-06 at 16 36 23](https://user-images.githubusercontent.com/6597467/132234249-bc0a90b3-cc85-4ebd-9592-15e1d219260f.png)

Reply:
Updated